specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -564,6 +564,48 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subcriptionId}/providers/Microsoft.Billing/usagePlan": {
```

Comment:
If usagePlan is a singleton for the subscription, this should be:
`/subscriptions/{subcriptionId}/providers/Microsoft.Billing/usagePlans/default`

By ARM standards, the current URL format would be for a collection, not a single response. usagePlan should be pluralized to usagePlans for consistency.